autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -1167,15 +1186,28 @@ impl DbClient for BigTableClientImpl {
         let mut req = ReadRowsRequest::default();
         req.set_table_name(self.settings.table_name.clone());
         req.set_filter({
-            let mut regex_filter = data::RowFilter::default();
+            let mut filter = data::RowFilter::default();
+
+            let mut chain = data::RowFilter_Chain::default();
+            let mut filter_chain = RepeatedField::default();
+
+            let mut row_filter = data::RowFilter::default();
             // channels for a given UAID all begin with `{uaid}#`
             // this will fetch all messages for all channels and all sort_keys
-            regex_filter.set_row_key_regex_filter(
+            row_filter.set_row_key_regex_filter(
                 format!("^{}#[^#]+#01:.+", uaid.simple())
                     .as_bytes()
                     .to_vec(),
             );
-            regex_filter
+
+            let time_filter = timestamp_filter()?;
+            filter_chain.push(time_filter);
+
+            filter_chain.push(row_filter);
```

Comment:
We definitely want the timestamp filter to be the last in the chain (like you have in the other method): https://cloud.google.com/bigtable/docs/using-filters#limiting-filters

```suggestion
            filter_chain.push(row_filter);

            filter_chain.push(time_filter);
```